Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,523 @@
+    {{476, 0x450F}, {"Food Fight (Virtual)", Game::TrapTeam, Element::Life, Type::Skylander}},
+    {{477, 0x3000}, {"High Five", Game::TrapTeam, Element::Life, Type::Skylander}},
+    {{478, 0x3000}, {"Krypt King", Game::TrapTeam, Element::Undead, Type::TrapMaster}},
+    {{478, 0x3402}, {"Krypt King (Nitro)", Game::TrapTeam, Element::Undead, Type::TrapMaster}},
+    {{479, 0x3000}, {"Short Cut", Game::TrapTeam, Element::Undead, Type::TrapMaster}},
+    {{480, 0x3000}, {"Bat Spin", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{481, 0x3000}, {"Funny Bone", Game::TrapTeam, Element::Undead, Type::Skylander}},
+    {{482, 0x3000}, {"Knight Light", Game::TrapTeam, Element::Light, Type::TrapMaster}},
+    {{483, 0x3000}, {"Spot Light", Game::TrapTeam, Element::Light, Type::Skylander}},
+    {{484, 0x3000}, {"Knight Mare", Game::TrapTeam, Element::Dark, Type::TrapMaster}},
+    {{485, 0x3000}, {"Blackout", Game::TrapTeam, Element::Dark, Type::Skylander}},
+    {{502, 0x3000}, {"Bop", Game::TrapTeam, Element::Earth, Type::Mini}},
+    {{503, 0x3000}, {"Spry", Game::TrapTeam, Element::Magic, Type::Mini}},
+    {{504, 0x3000}, {"Hijinx", Game::TrapTeam, Element::Undead, Type::Mini}},
+    {{505, 0x3000}, {"Terrabite", Game::TrapTeam, Element::Earth, Type::Mini}},
+    {{506, 0x3000}, {"Breeze", Game::TrapTeam, Element::Air, Type::Mini}},
+    {{507, 0x3000}, {"Weeruptor", Game::TrapTeam, Element::Fire, Type::Mini}},
+    {{507, 0x3402}, {"Weeruptor (Eggsellent)", Game::TrapTeam, Element::Fire, Type::Mini}},
+    {{508, 0x3000}, {"Pet Vac", Game::TrapTeam, Element::Air, Type::Mini}},
+    {{508, 0x3402}, {"Pet Vac (Power Punch)", Game::TrapTeam, Element::Air, Type::Mini}},
+    {{509, 0x3000}, {"Small Fry", Game::TrapTeam, Element::Fire, Type::Mini}},
+    {{510, 0x3000}, {"Drobit", Game::TrapTeam, Element::Tech, Type::Mini}},
+    {{514, 0x3000}, {"Gill Runt", Game::TrapTeam, Element::Water, Type::Mini}},
+    {{519, 0x3000}, {"Trigger Snappy", Game::TrapTeam, Element::Tech, Type::Mini}},
+    {{526, 0x3000}, {"Whisper Elf", Game::TrapTeam, Element::Life, Type::Mini}},
+    {{540, 0x1000}, {"Barkley (Sidekick)", Game::Giants, Element::Life, Type::Mini}},
+    {{540, 0x3000}, {"Barkley", Game::TrapTeam, Element::Life, Type::Mini}},
+    {{540, 0x3402}, {"Barkley (Gnarly)", Game::TrapTeam, Element::Life, Type::Mini}},
+    {{541, 0x1000}, {"Thumpling (Sidekick)", Game::Giants, Element::Water, Type::Mini}},
+    {{541, 0x3000}, {"Thumpling", Game::TrapTeam, Element::Water, Type::Mini}},
+    {{542, 0x1000}, {"Mini Jini (Sidekick)", Game::Giants, Element::Magic, Type::Mini}},
+    {{542, 0x3000}, {"Mini Jini", Game::TrapTeam, Element::Magic, Type::Mini}},
+    {{543, 0x1000}, {"Eye-Small (Sidekick)", Game::Giants, Element::Undead, Type::Mini}},
+    {{543, 0x3000}, {"Eye-Small", Game::TrapTeam, Element::Undead, Type::Mini}},
+    {{1000, 0x2000}, {"Boom Jet", Game::SwapForce, Element::Air, Type::Swapper}},
```

Comment:
I think you have missed adding (Top) and (Bottom) to the swappers

Reply:
Yup. Don't know when and why I removed those